SpatialConvolutionMM.lua

```diff
@@ -70,19 +70,49 @@ function SpatialConvolutionMM:updateOutput(input)
       self.padding = nil
    end
    input = makeContiguous(self, input)
-   return input.nn.SpatialConvolutionMM_updateOutput(self, input)
+   input.THNN.SpatialConvolutionMM_updateOutput(
+      input:cdata(),
+      self.output:cdata(),
+      self.weight:cdata(),
+      self.bias:cdata(),
+      self.finput:cdata(),
+      self.kW, self.kH,
+      self.dW, self.dH,
+      self.padW, self.padH
+   )
```

Comment:
how are you going to add `groups` here, by adding a for loop in lua or in C side?